crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -103,17 +103,30 @@ mod get_transaction_by_hash_integration_tests {
 
         // We need to flatten the ABI into a string first
         let flattened_class = contract_artifact.flatten().unwrap();
-        let result = account
+        let declare_result = account
             .declare(Arc::new(flattened_class), compiled_class_hash)
             .nonce(FieldElement::ZERO)
             .max_fee(FieldElement::from_hex_be("0xde0b6b3a7640000").unwrap()) // Specified max fee of 10^18 to declare v2 transaction, can be removed once fee estimation will work
             .send()
             .await;
 
-        assert_eq!(
-            result.unwrap().transaction_hash,
-            FieldElement::from_hex_be(DECLARE_V2_TRANSACTION_HASH).unwrap()
-        );
+        let result = devnet
```

Comment:
This is just a fix for the test in get_transaction_by_hash_integration_tests, sorry that it's not a separate PR.